BRANCHES.md

```diff
@@ -0,0 +1,55 @@
+Branches used in this repo
+==========================
+
+The document explains the branching structure that we are using in the git for VFSForGit repository as well as the forking strategy that we have adopted for contributing.
+
+Repo Branches
+-------------
+
+1. master
+    
+    This will track the git for windows repository master branch
+
+2. gvfs

```

Comment:
Why have we not renamed this to 'vfs' or something?

Reply:
Haven't gotten around to renaming these.  There are builds that will need to be updated.  There are tags and releases that are using 'gvfs'.  Not sure if we would want to change those, especially the releases.  The version for git is determined using the tags. @dscho, I'm sure you have automation around these branch names and tags, how much work is it to change this?  We could add the new names and have both while we get things updated.

Reply:
> I'm sure you have automation around these branch names and tags, how much work is it to change this?

I think the most important automation around this is the `GIT-VERSION-GEN` patches: febd85092824fdd716b5b9a9515ab721ed64dbeb and f357366f470ab397c72232721c9a4eef2268d698

Other than that: there are *plenty* of references in our internal builds. However, I think they mostly refer to the internal `gvfs/master` branch. Most of those internal builds are obsolete now, I think, e.g. the job that used to tell me when I need to synchronize the clean patches with the internal state, and the job that used to create and publish the installers (which I think we still want, but have to re-do?).

In short: there is not much automation that needs adjusting, so I would suggest changing it and seeing what breaks 😉